src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -462,6 +463,11 @@ private void BeforeAttribute()
                                  sym.Type != HtmlSymbolType.CloseAngle &&
                                  sym.Type != HtmlSymbolType.OpenAngle &&
                                  (sym.Type != HtmlSymbolType.ForwardSlash || !NextIs(HtmlSymbolType.CloseAngle)));
+
+                // capture whitespace after attribute name (if any)
+                whitespaceAfterAttributeName = ReadWhile(
+                    sym => sym.Type == HtmlSymbolType.WhiteSpace ||
+                    sym.Type == HtmlSymbolType.NewLine);
```

Comment:
I'm pretty sure this is just fine but we should validate that this newline and the previous whitespace symbol types can handle all the valid whitespace/newline characters defined by the spec for attributes:

```
- U+0020 SPACE
- U+0009 CHARACTER TABULATION (tab)
- U+000A LINE FEED (LF)
- U+000C FORM FEED (FF)
- U+000D CARRIAGE RETURN (CR)
```